spec/javascripts/test_spec.js

```diff
@@ -1,5 +1,31 @@
 describe("Basic testing", function() {
+  "use strict";
+
+ var fixture = loadFixtures('example.html');
+
   it("Basic Test", function () {
   expect(true).toBe(true);
   });
+
+  it("Checks if at zoom level 5 , all 7 markers are shown", function () {
+    //BlurredLocation1.map.setZoom(5);
+    expect($("#map1").children()[0].childNodes[3].childNodes.length).toBe(0) ;
```

Comment:
` expect($("#map1").children()[0].childNodes[3].childNodes.length).toBe(7) ;`

Reply:
Hi @rexagod , yes i did that purposely to tell you guys that in testing , markers are not getting loaded on map 😄 !